modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SetupDXP/index.js

```diff
@@ -20,39 +20,25 @@ import { actionTypes } from '../../context/reducer';
 import { getInitialDxpAdmin, steps } from '../../utils/constants';
 
 const SetupDXP = () => {
-	const [{ project, userAccount }, dispatch] = useContext(AppContext);
+	const [{ project }, dispatch] = useContext(AppContext);
 	const { errors, setFieldValue, touched, values } = useFormikContext();
 	const [baseButtonDisabled, setBaseButtonDisabled] = useState(true);
 
-	const [fetchSetupDXPInfo, { data }] = useLazyQuery(getSetupDXPInfo);
-
-	useEffect(() => {
-		fetchSetupDXPInfo({
-			variables: {
-				accountSubscriptionsFilter: `(${userAccount.accountBriefs
-					.map(
-						(
-							{ externalReferenceCode },
-							index,
-							{ length: totalAccountBriefs }
-						) =>
-							`accountKey eq '${externalReferenceCode}' ${index + 1 < totalAccountBriefs ? ' or ' : ' '
-							}`
-					)
-					.join('')}) and (contains(name, 'HA DR') or contains(name, 'Std DR'))`,
-				koroneikiAccountsFilter: `accountKey eq '${project.accountKey}'`
-			}
-		})
-	// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [project, userAccount]);
+	const { data } = useQuery(getSetupDXPInfo, {
+		variables: {
+			accountSubscriptionsFilter: `(accountKey eq '${project.accountKey}') and (contains(name, 'HA DR') or contains(name, 'Std DR'))`,
```

Comment:
@jairmedeiros , can we move these strings 'HA DR' and 'Std DR' to another place instead hard code?
Does it make sense?

Reply:
Using Object or Picklist? I can pass these strings to consts variables, but still hard coded.